client/controllers/curatorInbox/curatorInbox.coffee

```diff
@@ -63,8 +77,14 @@ Template.curatorInbox.onRendered ->
 
   @autorun =>
     if @ready.get()
+      if @selectedFeedId.get() in getCustomFeedArray()
```

Comment:
When the ready variable is false, the autorun won't see the reactive-variables inside this if statement and so won't react to them.